terraform/variables.tf

```diff
@@ -56,6 +56,16 @@ variable "mssql_sku_name" {
   default     = "Basic"
 }
 
+variable "application_insights_daily_data_cap_mb" {
```

Comment:
This is used to set `daily_data_cap_in_gb` so should be `gb` rather than `mb`

Can we also override in the production workspace_variables to `0.5`

Reply:
fixed